experimental/packages/opentelemetry-sdk-metrics-base/README.md

```diff
@@ -21,6 +21,57 @@ Please see the [version `0.27.0` README](https://github.com/open-telemetry/opent
 
 TODO: Add usage information for updated SDK
 
+## Installation of the Latest experimental version
+
+```bash
+npm install --save @opentelemetry/sdk-metrics-base
+```
+
+## Usage of the Latest experimental version
+
+The basic setup of the SDK can be seen as followings:
+
+```js
+const opentelemetry = require('@opentelemetry/api-metrics');
+const { MeterProvider } = require('@opentelemetry/sdk-metrics-base');
+
+// To create an instrument, you first need to initialize the Meter provider.
+// NOTE: The default OpenTelemetry meter provider does not record any metric instruments.
+//       Registering a working meter provider allows the API methods to record instruments.
+opentelemetry.setGlobalMeterProvider(new MeterProvider());
+
+// To record a metric event, we used the global singleton meter to create an instrument.
+const counter = opentelemetry.getMeter('default').createCounter('foo');
+
+// record a metric event.
+counter.add(1, { attributeKey: 'attribute-value' });
+```
+
+In conditions, we may need to setup an async instrument to observe costly events:
+
+```js
+// Creating an async instrument, similar to synchronous instruments
+const observableCounter = opentelemetry.getMeter('default')
+  .createObservableCounter('observable-counter');
+
+// Register a single-instrument callback to the async instrument.
+observableCounter.addCallback(async (observableResult) => {
+  // ... do async stuff
+  observableResult.observe(1, { attributeKey: 'attribute-value' });
```

Comment:
What happens if you call observe more than once? Is this allowed? Not sure if there is a usecase for that but i'm just curious

Reply:
It's allowed to call `observe` more than once, and with the identical attributes, the last observation of that particular attribute is preserved.